environments/mcdTransformers.toml

```diff
@@ -910,6 +917,10 @@
     deployed = 10742907
+  [contract.MCD_DOG_1_x_x]
```

Comment:
🚧 We'll need to update these contract details once the Dog contract is deployed. To update:
- the contract name MCD_DOG_1_x_x
- the contract address
- the ABI - though this ABI is likely correct, we should use the ABI that is posted when the contract is deployed to make sure that the right complier version was used,etc
- deployed block number